src/zoid/button/util.js

```diff
@@ -7,14 +7,29 @@ export function validateScopes(scopes : $ReadOnlyArray<string>) {
         throw new Error(`Expected props.scopes element to be non-empty`);
     }
 }
+export function validateRedirectUrl(returnurl : string) {
+    if (returnurl === undefined || returnurl.length === 0) {
+        throw new Error(`Expected returnurl to be set`);
+    }
+}
 
 
 export function validateResponseType(responseType : string) {
-    const RESPONSE_TYPES : $ReadOnlyArray<string> = [ 'code', 'code+token', 'code+token+id_token', 'code+id_token' ];
+    const RESPONSE_TYPES : $ReadOnlyArray<string> = [ 'code', 'code+token', 'code+token+id_token', 'code+id_token', 'id_token' ];
```

Comment:
I don't see all these values in the spec at https://github.com/paypal/paypal-sdk-spec/pull/55 -- please make sure this matches the spec (either remove values here, or add them in the spec)

Reply:
i removed it